lib/sails-migrations/helpers/sails_integration.js

```diff
@@ -48,8 +49,14 @@ SailsIntegration.getSailsConfig = function (modulesPath, sails) {
       moduleName = dbConfig.module;
       break;
     case "0.10":
-      defaultAdapterName = sails.config.models.connection;
-      dbConfig = sails.config.connections[defaultAdapterName];
+      if (args.connection) {
```

Comment:
lets make this cleaner,
could you please reformat this to be async and return an error in the correct place, so that we'll not have to do the instanceOf error check?

thanks,
Danni